docs/reference/utilities.rst

```diff
@@ -1,4 +1,32 @@
-.. automodule:: stravalib.util.limiter
-   :members:
-   :undoc-members:
-   :show-inheritance:
\ No newline at end of file
+======================================================
+Limiter - in Utility Submodule: Functions and Classes
+======================================================
+.. currentmodule:: stravalib.util.limiter
+
+This module provides a mixture of helpers to support rate limiting
+and also functions for conversion?
+
+**TODO:** look into whether total_seconds relates to limiter actions or
```

Comment:
there was one random function here and i didn't look closely to see if it was related to seconds in terms of timing the API return vs something else..

Reply:
That function `total_seconds()` seems to be only in support of python <2.7, so I think it can be removed.